rh/email_dhcr.py

```diff
@@ -0,0 +1,31 @@
+from django.core.mail import send_mail
+from project import common_data
+
+RH_EMAIL_TEXT = common_data.load_json("rh.json")
+
+DHCR_EMAIL_SENDER_ADDRESS = 'support@justfix.nyc'
+
+DHCR_EMAIL_RECIPIENT_ADDRESSES = ['rentinfo@nyyshcr.org', 'e8x9f4k1h2i3e6p3@justfixnyc.slack.com']
```

Comment:
Hmm, we should make this an environment variable, so that doing this on staging doesn't actually send the email to DHCR, and so we can change the slack address without having to modify the codebase (or reveal the email to the public for that matter).

You can do that by:

1. [Adding a new property to `justfix_environment.py`](https://github.com/JustFixNYC/tenants2/commit/bfacfff71fb92913c8c74cc7211bf8df5f313c0f#diff-c24ddcf5a5c5d3fa5b5e9128e12206c2R199-R209)

2. [Exposing the property in `settings.py`](https://github.com/JustFixNYC/tenants2/commit/bfacfff71fb92913c8c74cc7211bf8df5f313c0f#diff-9ca8db372ff7b61122729b2518307c66R377)

3. [Providing a default value for the testing environment in `settings_pytest.py`](https://github.com/JustFixNYC/tenants2/commit/bfacfff71fb92913c8c74cc7211bf8df5f313c0f#diff-8c032c58723e83ce937e5638daed2563R38)

You might want to document the environment variable to be a string, but use Python string's `split()` method (similar to the JavaScript one) to turn it into a list or something...

Reply:
awesome! just tried that out, let me know if I got that right

Reply:
https://github.com/JustFixNYC/tenants2/pull/891/commits/f0f72005186de405d3ec92ca62e74a4ee9a451d2